omniscidb/Tests/CostModel/CostModelTest.cpp

```diff
@@ -32,6 +34,25 @@ class DataSourceTest : public DataSource {
   }
 };
 
+bool doubleEquals(double x, double y) {
+  const double EPS = 0.0001;
+  return std::abs(x - y) < EPS;
+}
+
+class LinearRegressionTest : public LinearRegression {
+ public:
+  LinearRegressionTest(std::vector<Detail::Measurement>& measurement)
+      : LinearRegression(measurement) {}
+
+  void equalsWs(arma::vec wExpected) {
+    ASSERT_EQ(w.size(), wExpected.size());
+
+    for (size_t i = 0; i < w.size(); i++) {
```

Comment:
For vector comparisons and such, please consult with `TestHelpers.h` and `ArrowTestHelpers.h` first.

Reply:
Should i add comparison function for armadillo vectors?

Reply:
This is FYI, since you have a single smoke test there's no need for it to be in the header.